src/main/resources/locales/fr-FR.yml

```diff
@@ -0,0 +1,6 @@
+islandfly:
+  fly-outside-alert: "Vous êtes hors de votre île, vous ne pourrez plus voler dans [timeout] secondes."
+  disable-fly: "Votre mode volant vient d'être désactivé."
+  enable-fly: "Votre mode volant vient d'être activé."
```

Comment:
A better translation for the two above lines would be:
```
disable-fly: "Vous ne pouvez plus voler."
enable-fly: "Vous pouvez désormais voler."
```
Mais là encore, je te laisse le choix 😛.

Reply:
I changed these two lines,.

Effectivement sa me parait mieux :D